app/components/events/index.js

```diff
@@ -0,0 +1,38 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action  } from '@ember/object';
+
+export default class EventsComponents extends Component{
+
+	/** variable for data*/
+
+	//meeting variable for all meeting related details like meeting links
+	meeting = {
+		link: 'https://www.airmeet.com/e/57d5b2b0-c7a7-11ea-a4ea-fd3d36d2c7f6'
```

Comment:
I like that you ensured extracting hardcoded values and trying to keep your hbs as reusable as possible 
Kudos 🎉